<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelWidget</class>
 <widget class="QWidget" name="LevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
border: none !important;

vertical {
	border: 2px solid grey;
	background: #32CC99;
	width: 15px;
	margin: 20px 0px 20px 0px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hideButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>18</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: #F1F3F5;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
}</string>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>396</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(40, 41, 42, 240);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>18</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: #A1A3A5;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
}</string>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>18</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: #A1A3A5;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #FF4345;
}

QPushButton:pressed
{
	color: #990000;
}</string>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>18</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: #A1A3A5;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
}</string>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>listWidget</zorder>
  <zorder>widget</zorder>
  <zorder>hideButton</zorder>
  <zorder>randomButton</zorder>
  <zorder>resetButton</zorder>
  <zorder>restartButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
